rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -51,6 +52,11 @@
     Object execute(final OperationChain opChain);
 
     @POST
+    @Path("/operation")
+    @ApiOperation(value = "Performs the given operation on the graph", response = Object.class)
+    Object execute(final Operation operation);
+
+    @POST
     @Path("/chunked")
```

Comment:
For consistency, do we want to add a chunked implementation of the "execute single operation" endpoint?